public/js/cal.js

```diff
@@ -8,7 +8,7 @@ var selectedRoom;
 var eventTitle;

```

Comment:
You might consider wrapping this whole thing in an IIFE; as it stands, each of these variables and functions become global and could easily be accidentally overwritten by someone else's bit of code.